components/rhs_root_post/rhs_root_post.jsx

```diff
@@ -99,6 +99,8 @@ export default class RhsRootPost extends React.PureComponent {
          */
         isPostBeingEdited: PropTypes.bool,
         isMobileView: PropTypes.bool.isRequired,
+
+        filePreviews: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
could you please add a short description of the prop here?

Reply:
I added comment.